src/main/java/pl/pelikan/ex_27/model/Employee.java

```diff
@@ -0,0 +1,82 @@
+package pl.pelikan.ex_27.model;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDate;
+
+@Entity
+public class Employee {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String firstName;
+    private String lastName;
+    @DateTimeFormat(pattern = "yyyy-MM-dd")
+    private LocalDate birthDate;
+    private String email;
+    private boolean isSignedUpToNewsletter;
```

Comment:
przedrostek "is" jest w nazwach metod. Jako pole powinno to być "signedUpToNewsletter", a poprawniej po angielsku signedUpForNewsletter :P